src/routes/docs/products/sites/domains/+page.markdoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: article
+title: Domains
+description: Discover how domains can be managed for an Appwrite Site
+---
+
+Each deployed site can have its own domain, which can be Appwrite-generated or custom. You can use this domain to consume web apps deployed on Appwrite Sites. Appwrite generates TLS certificates to enforce HTTPS on all Appwrite Sites domains. These domains are safe to use and access in production.
+
+[Learn about Sites development >](/docs/products/sites/develop)
+
+## Generated domains
+
+1. In the Appwrite Console's sidebar, click **Sites**.
+2. Under the **Domains** tab, you'll find the domain generated by Appwrite.
+
+The domain usually has this format.
+
+```bash
+https://64d4d22db370ae41a32e.sites.appwrite.global
```

Comment:
Confirm with CHristy, I think all domains will become `.network` on sites.

For functions, .network will also be available, but I think they will also support .run, for region-specific run. Check that with christy too; we may want that rework as separate PR, but for Sites, let's have new docs accurate

Reply:
Will check on this

Reply:
it will.